specifications/Paxos/MCVotingApalache.tla

```diff
@@ -0,0 +1,32 @@
+--------------------------- MODULE MCVotingApalache -------------------------------
```

Comment:
Elsewhere, we recently discussed TLA+ naming conventions.   `MC*Apalache` is really verbose.  Perhaps, we want to adopt and encourage a less verbose convention?  IIRC, Apalache already prefixes some operators with `Apa`.  So how about `ApaMCVoting` or `APAMCVoting`?  Would then even be the same number of chars if users want to do `APAMCVoting` and `TLCMCVoting`.  On the other hand, we might just as well drop `MC`.

Reply:
The `MC` prefix seems superfluous in that case. In this particular example, do you have a suggestion for the name of the specification module? `Voting2.tla` and `ApaVoting2.tla`?